### ISSUE TITLE
Add Linux Arm64 support 

### ISSUE THREAD

**Issue by odidev** [fonttools/skia-pathops#48]
Hi Team,  
I have been working on [fonttools](https://github.com/fonttools/fonttools) for arm64 and it has a dependency on skia-pathops.  

Tried installing it through [pypi](https://pypi.org/project/skia-pathops/#files) source code, but it looks like skia-pathops is not available for arm64 currently and so got this error: 
```
ERROR: Failed building wheel for skia-pathops 

Failed to build skia-pathops 

ERROR: Could not build wheels for skia-pathops which use PEP 517 and cannot be installed directly 
```
Do you have any plans of releasing it for arm64? 
 
It would be very helpful if skia-pathops will support arm64 too.

**Comment by alerque** [comment]
Does chromium build for arm64?

**Comment by eliheuer** [comment]
I'm having the same issue on an M1 mac 11.6 desktop, the same error from above,
```
ERROR: Could not build wheels for skia-pathops which use PEP 517 and cannot be installed directly 
```
is causing `pip3 install gftools` to fail for me: https://github.com/googlefonts/gftools/issues/415

**Comment by eliheuer** [comment]
@alerque as far as I know yes: https://chromium.googlesource.com/chromium/src.git/+/main/docs/mac_arm64.md

**Comment by anthrotype** [comment]
Let's keep this issue open, as I only partly addressed the request in #50: we now build universal2 wheels for _mac_, which include support for the `arm64` architecture, but not yet for _linux_ `arm64` (also known as `aarch64`) architecture.
That seems to be more involved. Looks like the skia buildtools don't come with a precompiled `gn` for linux-aarch64.
I'll continue to investigate another time when I'll get the chance.
https://github.com/fonttools/skia-builder/pull/4

**Comment by anthrotype** [comment]
@odidev skia-pathops v0.7.1 now ships with linux-aarch64 (arm64) wheels as well, please test, thanks!
https://pypi.org/project/skia-pathops/0.7.1/#files
